feat(docs): Add docusaurus plugin to autogerate llms.txt

critesjosh opened the pull request:
Adds a plugin to the docs to autogenerate an llms.txt file and an llms-full.txt file to be published with the docs site according to the standards defined at https://llmstxt.org/. 

Also adds page descriptions to pages that are missing them, since this helps with generating the llms.txt.

Note that this is set up to generate the file for the first versioned defined in the versions.json file, which should be the default version for the docs site.

Also removes an outdated sandbox features page.

closes: https://github.com/AztecProtocol/dev-rel/issues/597

critesjosh commented:
looks like some page descriptions still need to be updated

critesjosh commented on docs/docs/developers/guides/smart_contracts/writing_contracts/call_functions.md:
removing this because it is the same page as docs/docs/developers/guides/smart_contracts/writing_contracts/call_contracts.md

alejoamiras commented on docs/docs/developers/guides/local_env/how_to_debug.md:
```diff
@@ -138,12 +139,12 @@ Current limits that trigger `7009 - ARRAY_OVERFLOW`:
-| `next available leaf index mismatch` | Tree corruption | Verify tree updates are sequential |
-| `Public call stack size exceeded` | Too many public calls | Reduce public function calls |
-| `Failed to publish block` | L1 submission failed | Check L1 connection and gas |
+| Error                                | Cause                 | Solution                                   |
+| ------------------------------------ | --------------------- | ------------------------------------------ |
+| `tree root mismatch`                 | State inconsistency   | Restart sandbox or check state transitions |
+| `next available leaf index mismatch` | Tree corruption       | Verify tree updates are sequential         |
+| `Public call stack size exceeded`    | Too many public calls | Reduce public function calls               |
+| `Failed to publish block`            | L1 submission failed  | Check L1 connection and gas                |
```
Is this claude's work on formatting or yours? just want to double check that all format changes are intended (applies to some other files).

critesjosh replied:
i believe its my formatter, not claude

alejoamiras reviewed (approved):
Other than some formatting issue (some changes from `'` to `"` and some tables) the PR looks straight forward, and pretty kewl 🔥

critesjosh committed:
feat(docs): Add docusaurus plugin to autogerate llms.txt

Adds a plugin to the docs to autogenerate an llms.txt file and an llms-full.txt file to be published with the docs site according to the standards defined at https://llmstxt.org/.

Also adds page descriptions to pages that are missing them, since this helps with generating the llms.txt.

Note that this is set up to generate the file for the first versioned defined in the versions.json file, which should be the default version for the docs site.

Also removes an outdated sandbox features page.
